src/commands/VoidTrader.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const Command = require('../Command.js');
+
+/**
+ * Displays the currently active Invasions
+ */
+class VoidTrader extends Command {
+  /**
+   * Constructs a callable command
+   * @param {Genesis} bot  The bot object
+   */
+  constructor(bot) {
+    super(bot, 'ondemand.voidtrader', 'trader', 'Display the currentstatus of the void trader');
```

Comment:
The command was baro in the previous version, I don't think it should be changed in order not to confuse people